helpers/Breadcrumbs.php

```diff
@@ -145,20 +145,50 @@ public static function deliveryMonitoringAll(core_kernel_classes_Resource $testC
     }
 
     /**
-     * Create breadcrumb for Delivery::testTaker
+     * Create breadcrumb for Delivery::manage
      *
      * @param core_kernel_classes_Resource $testCenter
      * @param core_kernel_classes_Resource $delivery
      * @param array $deliveries
      * @return array
      */
-    public static function deliveryTestTaker(core_kernel_classes_Resource $testCenter, core_kernel_classes_Resource $delivery)
+    public static function deliveryManage(core_kernel_classes_Resource $testCenter, core_kernel_classes_Resource $delivery)
+    {
+        return array(
+            'id' => 'deliveryManager',
+            'url' => _url('manage', 'Delivery', null, array('testCenter' => $testCenter->getUri(), 'delivery' => $delivery->getUri())),
+            'label' => __('Manage'),
```

Comment:
I would call the action "Manage Test Takers" to make it more consistent with the other action "Add Test Takers".